### DISCUSSION TITLE
Where can I get the web version of the bridge source code

### DISCUSSION THREAD

**Discussion by LYZQ2107** [bridge-core/bridge.#449] [Q&A] [answered]
Where can I get the web version of the bridge source code (many Chinese developers can't access the web version of the bridge because the servers are located in other countries)?

**Comment by outercloudstudio** [comment, marked as answer]
This is the bridge v1 repository. It's outdated. The newest one is the editor repository which has the web compatible code.

**Reply by LYZQ2107** [reply]
thanks
